api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -54,6 +54,30 @@ private Relationships() {
         owns
     }
 
+
+    /**
+     * The list of possible relationship (aka edges) direction. Relationships are not bidirectional.
+     */
+    public enum Direction {
+        /**
+         * Relative to the current position in the Hawkular pipeline, this value expresses such relationships
+         * that has me (the entity(ies) on the current pos) as a target(s).
```

Comment:
Isn't this actually the other way round?
I was assuming that `...relationships(outgoing).named(defines).resources().entities()` would get me all the resources that are *targets* of defines relationships going out from "me" - i.e. "me" is the source of the relationships.